shared/js/ui/templates/privacy-options.es6.js

```diff
@@ -36,5 +36,17 @@ module.exports = function () {
             </p>
         </li>
     </ul>
+    <ul class="default-list">
```

Comment:
Missing adding this behind the feature flag.

Reply:
I'm not sure we have a feature-flag system for things like this, but how about we comment out this block with a TODO note to put it back when finishing YouTube CTL?